flask_potion/__init__.py

```diff
@@ -182,8 +193,7 @@ def add_route(self, route, resource, endpoint=None, decorator=None):
 
         view = self.output(view_func)
```

Comment:
Ok, blueprints made this a bit more complicated and endpoint was also not the right approach to testing if a route belonged to a schema. I went ahead and pushed a change that should resolve the issue.